_shared/project/pyproject.toml

```diff
@@ -59,6 +59,12 @@ omit = [
     {% if cookiecutter.get("_directory") == "pyapp" %}
     "*/{{ cookiecutter.package_name }}/app.py",
     {% endif %}
+    {% if cookiecutter.get("_directory") == "pyramid-app" %}
+    "{{ cookiecutter.package_name }}/pshell.py",
+    {% endif %}
+    "tests/pytest_plugins/cookiecutter.py",
+    "tests/unit/pytest_plugins/cookiecutter.py",
+    "tests/functional/pytest_plugins/cookiecutter.py",
```

Comment:
This is just adding some files to coverage's `omit` list: the `pshell.py` file (I don't think there's any value in unit testing this, it's just a startup file for `make shell`), and the three `pytest_plugins/cookiecutter.py` files: these files belong to the cookiecutter not to the project, so for example if one of these files contained a pytest fixture that the project happened not to be using then you would get coverage misses but the project wouldn't be able to do anything about it since the coverage misses are in a cookiecutter-generated file.